tflite2onnx/op/reshape.py

```diff
@@ -72,13 +73,45 @@ def parse(self):
 
         self.setParsed()
 
+    def fakeTranspose(self):
```

Comment:
What about naming the function as something like `preserveInputSpatialSemantic`? We named `fakeBroadcast` in `Binary` because we were doing something like a _fake broadcasting_.